web/src/components/starter-kits/StepInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { Step } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+export const StepInfo = (props: { step: Step; taskNames: string[] }): ReactElement => {
+  return (
+    <div className="flex-basis-100">
+      <div className="margin-top-05 margin-bottom-2">
+        <span className="border-2px border-secondary-vivid-dark text-secondary-vivid-dark radius-lg padding-x-1 padding-y-05">
+          <b>STEP {props.step.stepNumber}</b>
+        </span>
+      </div>
+      <Heading level={3} styleVariant="h4">
```

Comment:
[sand] On the other hand, I don't believe the step names should be headings, since they don't really define a section of content; the steps do. Or rather, the heading should read as "Step 1: Plan Your Business" somehow.

Reply:
I do have a couple ideas of how to potentially achieve this with varying levels of complexity if @adidner prefers this approach